src/components/SlateEditor/RichTextEditor.jsx

```diff
@@ -101,29 +86,36 @@ const RichTextEditor = class extends React.PureComponent {
       plugins,
       schema,
       value,
-      ...rest
+      renderBlock,
+      renderInline,
+      renderMark,
+      submitted,
     } = this.props;
+
     return (
       <article>
-        <div data-cy="slate-editor" css={slateEditorDivStyle}>
-          <Editor
-            {...classes('content', undefined, className)}
-            {...rest}
-            id={id}
-            onKeyDown={this.onKeyDown}
-            ref={this.editorRef}
-            value={value}
-            name={name}
-            schema={schema}
-            onChange={this.onChange}
-            slateStore={this.state.slateStore}
-            plugins={plugins}
-            placeholder={placeholder}
-            onBlur={onBlur}
-            data-cy={dataCy}
-          />
-          {children}
-        </div>
+        <SlateProvider isSubmitted={submitted}>
+          <div data-cy="slate-editor" css={slateEditorDivStyle}>
+            <Editor
+              {...classes('content', undefined, className)}
+              renderBlock={renderBlock}
+              renderInline={renderInline}
```

Comment:
Da jeg la til renderInline og renderMark her så crashet toolbaren

Reply:
@gunnarvelle renderMark funka fint og fiksa italic og slik.
Noen som vet hva renderInline skal gjøre? For om jeg legger den til å crasher lenke og liste og matte osv

Reply:
renderInline rendrer spans. Trur ikkje det er så mange av dei, men dei burde jo være med.